monitor/grafana/zeebe.json

```diff
@@ -48,8 +48,8 @@
     },
     {
       "type": "panel",
-      "id": "table",
-      "name": "Table",
+      "id": "table-old",
```

Comment:
Is it necessary?

Reply:
I don't know what that is but I didn't change it by hand. I exported the dashboard in the way described in the README